src/main/java/Duke.java

```diff
@@ -1,10 +1,72 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+
 public class Duke {
+    private TaskList tasks;
+    private Storage storage;
+    private String fileName;
+    private Parser parser;
+    private Ui ui;
+
+    public Duke(String fileName) {
+        this.fileName = fileName;
+        storage = new Storage(fileName);
+        tasks = new TaskList(storage.loadTasks());
+        parser = new Parser();
+        ui = new Ui();
+        File file = new File(this.fileName);
+
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            Ui.printLines("File creation failed.");
+        }
+    }
+    
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        new Duke("src/main/data/tasks.txt").run();
```

Comment:
One recommendation I have is that the file for storage, in this case, tasks.txt, could be given a relative path such as "./data/tasks.txt", so that it would be more flexible in its environment. :)